parsimony/functions/multiblock/losses.py

```diff
@@ -687,6 +687,7 @@ def step(self, w, index):
                 all_lipschitz = False
                 break
             else:
+                print 'added stg'
```

Comment:
Python 2.7 format, so won't work in Python 3. This print statement should be removed.